setup.py

```diff
@@ -10,7 +10,20 @@
                     "docopt>=0.6",
                     "Jinja2>=2.7",
                     "PyYAML>=3.10",
-                    "python-dateutil>=2.2"]
+                    "python-dateutil>=2.2",
+                    # IPython dependencies:
```

Comment:
Well, I do not trust them enough to do that.

But if you wish I may fix that if you tell me which versions should I allow.